src/Components/StepTwo/StepTwo.js

```diff
@@ -1,63 +1,284 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import './StepTwo.scss';
+import { dataGet } from '../Axios/Axios';
```

Comment:
Please consider name of the function here. Maybe "getData" would be more descriptive, and clear? We have further in the code "setData", so that would be more consistent.